app/experimenter/templates/experiments/radio_field_label_inline.html

```diff
@@ -0,0 +1,17 @@
+<div class="form-group row">
+  <div class="col-3 text-right">
+    <label for="{{ field.id_for_label }}">
+      <strong>{{ field.label }}</strong>
+      <br>
+      <a target="_blank" rel="noreferrer noopener" href="{{ field.help_text }}">Help</a>
+    </label>
+  </div>
+  <div class="col-9">
+    <div class="row">
+      {{ field }}
+    </div>
+
+  </div>
+
+  {% include "experiments/field_errors_inline.html" with field=field %}
+</div>
```

Comment:
Again, needed a custom template to make an inline radio button that had a label on the left (as opposed to above).

Reply:
It took me a bit but yes, I see the other radio fields have the labels above the buttons.